lib/tmdb_handler.rb

```diff
@@ -166,23 +166,15 @@ def tmdb_handler_tv_series_autocomplete(query)
   end
 
   def tmdb_handler_tv_series(show_id)
```

Comment:
My goals here:
* remove all local instance variables from these two methods (& set controller instance variable equal to the method)
* change the methods to be more ruby conventional by having the last line be the important value being returned